frontend/src/components/Signup/Signup.css

```diff
@@ -0,0 +1,115 @@
+.signupBG {
```

Comment:
Can you use BEM naming for uniformity? So if my component were `Signup`, it would be something like `.signup__background`. In other words, `[component name]__[component part]--[some sort of flag (optional)]`